src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -160,15 +164,22 @@ const OptionRowLHN = (props) => {
                                 )
                             }
                             <View style={contentContainerStyles}>
-                                <DisplayNames
-                                    accessibilityLabel="Chat user display names"
-                                    fullTitle={optionItem.text}
-                                    displayNamesWithTooltips={displayNamesWithTooltips}
-                                    tooltipEnabled
-                                    numberOfLines={1}
-                                    textStyles={displayNameStyle}
-                                    shouldUseFullTitle={optionItem.isChatRoom || optionItem.isPolicyExpenseChat}
-                                />
+                                <View style={[styles.flexRow, styles.alignItemsCenter]}>
+                                    <DisplayNames
+                                        accessibilityLabel="Chat user display names"
+                                        fullTitle={optionItem.text}
+                                        displayNamesWithTooltips={displayNamesWithTooltips}
+                                        tooltipEnabled
+                                        numberOfLines={1}
+                                        textStyles={displayNameStyle}
+                                        shouldUseFullTitle={optionItem.isChatRoom || optionItem.isPolicyExpenseChat}
+                                    />
+                                    {optionItem.isChatRoom && (
```

Comment:
I am wondering if the check should be on `subtitle` here?

Reply:
Do you mean we should check if the `subtitle` is empty or not?

Reply:
yeah! I am not sure if the `subtitle` is set only for the workspace chat rooms. Can you confirm this please?

Reply:
I checked and `subtitle` is also set for the workspace chat itself (chat type is **policy expense chat**) and the `subtitle` text is always **Workspace**. You can look at below image where I tried to change the condition to check whether `subtitle` is empty or not.

![327485](https://user-images.githubusercontent.com/50919443/196027273-173ff1f9-33c4-41be-b2d2-f513ef62fb58.jpg)

And here is the code to decide the subtitle text.
https://github.com/Expensify/App/blob/e38df100d62d197689bb16888a3ac7859f157681/src/libs/ReportUtils.js#L278-L299

If you look at the code, chat room will have subtitle if it's a 
1. default room
2. user created policy room
3. **policy expense chat**

And if we take a look at below code to check if it's a chat room or not
https://github.com/Expensify/App/blob/e38df100d62d197689bb16888a3ac7859f157681/src/libs/ReportUtils.js#L194-L202

we can see that it's a chat room if it's a
1. default room
2. user created policy room

So, if we want to achieve the same result as image above, then we need to check the `subtitle` contents. Otherwise, I think checking `isChatRoom` is enough.

Reply:
So, what do you think? Should I check the subtitle or keep the current check with isChatRoom?

Reply:
Lets stick to what you have right now `isChatRoom`. My bad to create unnecessary confusion.

Reply:
Done